### ISSUE TITLE
Feature request, skip chapter 0

### ISSUE THREAD

**Issue by FantiXX2** [tachiyomiorg/tachiyomi#10140]
### Describe your suggested feature

Hey! Love the projet I've been reading on tachiyomi for Years! I just thought of adding a feature that would allow to skip chapter 0. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.14.7](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
Just mark it as read?

**Comment by FantiXX2** [comment]
> Just mark it as read?

Wdym?

**Comment by FantiXX2** [comment]
> Just mark it as read?

Doing it every time is boring dont you agree? It's like the feature to skip chapters you've already read, you could do it yourself. Beside, it would surely be something quite simple to prog.

**Comment by BrutuZ** [comment]
> Doing it every time is boring dont you agree?

I disagree. At best you'd be saving **literally** just a couple of taps per series!

**Comment by CrepeTF** [comment]
Not even a couple of taps though, you can just swipe on the chapter to mark it as read now
